src/tests/bounding_box_of cylinder.json

```diff
@@ -0,0 +1,56 @@
+{
+  "cylinder_1": {
+    "position" :[0,0,0],
+    "rotation_angle" : [0,0],
+    "dimension" : [1,1],
+    "expected_dimension_box" : [2.0,2.0,1.0]
+  },
+  "cylinder_2": {
+    "position" :[0,0,0],
+    "rotation_angle" : [0,45],
+    "dimension" : [1,1],
+    "expected_dimension_box" : [2.0, 2.122, 2.122]
+  },
+  "cylinder_3": {
+    "position" :[0,0,0],
+    "rotation_angle" : [45,0],
+    "dimension" : [1,1],
+    "expected_dimension_box" : [2.0,2.0,1.0]
+  },
+  "cylinder_4": {
+      "position" :[0,0,0],
+      "rotation_angle" : [90,45],
+      "dimension" : [1,1],
+      "expected_dimension_box" : [2.122, 2.0, 2.122]
+    },
+    "cylinder_5": {
+      "position" :[0,0,1],
+      "rotation_angle" : [30,30],
+      "dimension" : [0.5,2],
+      "expected_dimension_box" : [1.366,1.616,2.232]
+    },
+    "cylinder_6": {
+      "position" :[0,0,1],
+      "rotation_angle" : [0,-45],
+      "dimension" : [1,1],
+      "expected_dimension_box" : [2.0, 2.122, 2.122]
+    },
+    "cylinder_7": {
+      "position" :[0,0,1],
+      "rotation_angle" : [0,315],
+      "dimension" : [1,1],
+      "expected_dimension_box" : [2.0, 2.122, 2.122]
+    },
+    "cylinder_8": {
+      "position" :[0,0,1],
+      "rotation_angle" : [0,90],
+      "dimension" : [1,1],
+      "expected_dimension_box" : [2.0,1.0,2.0]
+    },
+    "cylinder_9": {
+      "position" :[0,0,1],
```

Comment:
Is it possible to change some of these positions something other than [0,0,0] and [0,0,1]? Just to make sure it works as expected.

Reply:
yes ,sure. just for find out, position is independent in this mechanism. because position of bounding box is exactly position of the shape.

Reply:
not a bad idea actually. Mehdi we need to always keep in mind that testcases must be extensive. We must not make assumptions. When designing testcases, we don't know that "position of bounding box is exactly position of the shape".